LICENSE

```diff
@@ -0,0 +1,13 @@
+   Copyright 2024 [name of copyright owner]
+
+   Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Are we sure we want to use Apache License? How is the plugin/PUT-Project licensed?

Reply:
It may stay. Vision Lab is also using Apache License. See: https://github.com/PUTvision/qgis-plugin-deepness/blob/devel/LICENSE